### ISSUE TITLE
Calibre-web can not update Calibre-Library-Data after changing meta data in Calibre-web 

### ISSUE THREAD

**Issue by abubaman** [janeczku/calibre-web#522]
I have updated all book's meta data(tags, author, series, rating) and it works fine.

But after I tried to update metadata(description) for one book, I found that I can see any changes in calibre-web but it doesn't apply to Calibre-Library.

So after that, I tried to add new book to Calibre-Library. but it didn't work. Calibre-web didn't change anything even though I clicked 'Re-Connect to Calibre DB' in admin page.

So do you have any idea to solve this problem? If you need further information please let me know.

**Comment by saitoh183** [comment]
I am also interested in how i should manage my library. Now that i moved everything to google drive and i am using the metadata monitoring, should i be adding books via calibre-web only or should i add them via calibre? Does doing it from one or the other matter?

**Comment by OzzieIsaacs** [comment]
"But after I tried to update metadata(description) for one book, I found that I can see any changes in calibre-web but it doesn't apply to Calibre-Library." -> calibre itself displays the data it sees at the start of the program. Its not expecting anybody else to change data, so its not rereading the data from the library. After closing and opening calibre the data should be updated in calibre.
Its not a good idea to change data in calibre and calibre-web in parallel, there is the posibility that it will crash the database at one time. I'm having good experience with having calibre-web running. Starting calibre, doing some database operations (adding books, editing metadata, etc.) and than closing calibre and working with calibre web.
Regarding the book not visible in calibre-web, could you please check if the book is visible after restarting calibre-web?

**Comment by OzzieIsaacs** [comment]
@saitoh183: Adding books with calibre will be difficult if you have the database on gdrive, I think calibre can't handle it, so the only way of adding books is via calibre-web, otherwise you have to sync books to gdrive by hand. 
There is a difference between adding books by calibre and calibre-web. Calibre creates for every book a metadata.opf. Changes in the books metadata (title, author, etc.) are written to this file and also written to the database (metadata.db). If you rebuild a database with calibre with books added by calibre-web, all changes will be lost, because calibre-web only writes the changes to its internal database. Whereas added books by calibre can be rebuild from the metadata.opf file. Creating metadata.opf can be triggered in calibre by using library button -> library maintenance -> status of metadatabackup -> queue all books to metadata backup (texts similar, I don't have english calibre-running). If you have further questions, or need more details, please open a new issue, because otherwise anybody gets confused, because post one and two have nothing to do with eachother.

**Comment by OzzieIsaacs** [comment]
No feedback for long time
